#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_fix-nostrip.dpatch by Thomas Günther <tom@toms-cafe.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes DEB_BUILD_OPTIONS=nostrip.

@DPATCH@
--- femon-1.1.3/Makefile
+++ femon-1.1.3/Makefile
@@ -87,9 +87,6 @@ all: libvdr-$(PLUGIN).so
 
 libvdr-$(PLUGIN).so: $(OBJS)
 	$(CXX) $(CXXFLAGS) -shared $(OBJS) -o $@
-ifndef FEMON_DEBUG
-	@$(STRIP) $@
-endif
 	@cp --remove-destination $@ $(LIBDIR)/$@.$(APIVERSION)
 
 dist: clean
